pyscriptjs/src/plugins/pyterminal.ts

```diff
@@ -6,8 +6,21 @@ import { getLogger } from '../logger';
 import { type Stdio } from '../stdio';
 import { InterpreterClient } from '../interpreter_client';
 
+type AppConfigStyle = AppConfig & { terminal?: boolean | 'auto'; docked?: boolean | 'docked' };
```

Comment:
> I see we were passing the terminal config directly into the configure

@FabioRosado what do you mean?

For context, the idea is that the config contains settings for the whole application and the plugins. But each plugin can define its own settings of course, so it seems correct to me that `pyterminal` augments the type locally.

Then, we could also have a broader discussion to decide whether it makes sense to type the individual settings of `AppConfig`: currently we try to type it as an object where each field has a type and you cannot access non-existent fields, but personally I see it more as a dictionary where you can store arbitrary stuff with arbitrary key. But this is OT here :)

Reply:
I just remembered that we talked about this on your PR haha
Was thinking that since these are core plugins we could include it, but again happy to leave it as it is 👍

Reply:
So, should I resolve this? 🤔

Reply:
yes :)